src/dlgTriggerEditor.cpp

```diff
@@ -7590,6 +7591,21 @@ void dlgTriggerEditor::slot_import()
 }
 
 void dlgTriggerEditor::doCleanReset()
+{
+    if (mCleanResetQueued) {
+        return;
+    }
+
+    mCleanResetQueued = true;
+
+    QTimer::singleShot(0, [=]() {
```

Comment:
This is harnessing the *run this code when everything else has been done and we are idle* usage of a `QTimer` isn't it - and in that context I *think* it makes sense.

Reply:
Yup!